pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -139,88 +141,28 @@ func (r *ReconcileNodeNetworkConfigurationPolicy) Reconcile(request reconcile.Re
 			// Return and don't requeue
 			return reconcile.Result{}, nil
 		}
+		conditionsManager.FailedToFindPolicy(err)
```

Comment:
This is wrong. The previous if is ran if it is not found (and it not being found is totally valid, someone was just faster in removing it). If we reach to this point, it is not that the policy does not exist (and cannot be found), it means we have some unexpected bad error. We should make it obvious. The log is good, but for the condition, we should be explicit that it is an unexpected error. (Also, how do we set the condition if there is no policy)

Reply:
You are right, conditions are at policy we cannot state at policy that it cannot be found at conditions :-)

Let's just remove this since there is nothing except for the log that we can do.